lib/active_merchant/billing/gateways/securion_pay.rb

```diff
@@ -92,8 +115,18 @@ def scrub(transcript)
 
       private
 
+      def create_customer_add_card(authorization, options)
+        post = {}
+        post[:email] = options[:email]
+        post[:description] = options[:description]
+        post[:card] = authorization
+        post[:metadata] = {}
+        post[:metadata][:chargeId] = authorization
+        commit('customers', post, options)
+      end
+
       def add_customer(post, payment, options)
-        post[:customer] = options[:customer] if options[:customer]
+        post[:customerId] = options[:customer_id] if options[:customer_id]
```

Comment:
Is the `:customer` field name not supported at all (meaning this never worked)?

If so, we can be a little more liberal in introducing this breaking change.